platform-sdk/swirlds-common/src/main/java/com/swirlds/common/sequence/map/StandardSequenceMap.java

```diff
@@ -36,22 +34,24 @@ public class StandardSequenceMap<K, V> extends AbstractSequenceMap<K, V> {
     /**
      * Construct a {@link SequenceMap}.
      *
-     * @param firstSequenceNumberInWindow
-     * 		the lowest allowed sequence number
-     * @param sequenceNumberCapacity
-     * 		the number of sequence numbers permitted to exist in this data structure. E.g. if
-     * 		the lowest allowed sequence number is 100 and the capacity is 10, then values with
-     * 		a sequence number between 100 and 109 (inclusive) will be allowed, and any value
-     * 		with a sequence number outside that range will be rejected.
-     * @param getSequenceNumberFromKey
-     * 		a method that extracts the sequence number from a key
+     * @param firstSequenceNumberInWindow the lowest allowed sequence number
+     * @param sequenceNumberCapacity      the number of sequence numbers permitted to exist in this data structure. E.g.
+     *                                    if the lowest allowed sequence number is 100 and the capacity is 10, then
+     *                                    values with a sequence number between 100 and 109 (inclusive) will be allowed,
+     *                                    and any value with a sequence number outside that range will be rejected.
+     * @param allowExpansion              if true, then instead of rejecting elements with a sequence number higher than
+     *                                    the allowed by the current capacity, increase capacity and then insert the
+     *                                    element. Does not expand if the sequence number is too low to fit in the
+     *                                    current capacity.
+     * @param getSequenceNumberFromKey    a method that extracts the sequence number from a key
      */
     public StandardSequenceMap(
```

Comment:
You could add another constructor without `allowExpansion` that defaults to `false`. It would remove the need to change so many other invocations to pass in `false`.

Reply:
I've added the requested constructor, and fixed all the pre-existing locations that don't want to permit expansion.